spec/features/admin/affirm_payment_method_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+RSpec.describe "Affirm payment method admin" do
+  stub_authorization!
+
+  before(:each) do
+    visit spree.admin_path
+    click_link "Settings"
+  end
+
+  context "admin creating a new payment method" do
```

Comment:
Right, but the only valid test (if we were to write one) is verifying that it is set in the configuration. Adding a test dependency on the admin doesn't make sense because it's out of scope of what the extension accepts responsibility for. The admin may choose to present payment methods to the user in a completely different way, but that's the responsibility of the admin, we shouldn't have to verify how it works if we're not changing it.

If we had deface overrides or something that would make sense. Even further if I'm following correctly, this feature test is why you can't support solidus versions < 1.3 right?